test/py/ganeti.utils.x509_unittest.py

```diff
@@ -78,10 +78,11 @@ class TestGetX509CertValidity(testutils.GanetiTestCase):
   def setUp(self):
     testutils.GanetiTestCase.setUp(self)
 
-    pyopenssl_version = distutils.version.LooseVersion(OpenSSL.__version__)
+    pyopenssl_version = version.parse(OpenSSL.__version__)
+    desired_pyopenssl_version = version.parse("0.7")
```

Comment:
maybe switch the name from `desired` to `minimal`, this would better represent the need `>=`?

Reply:
Thanks for the hint.
minimal_pyopenssl_version is a more meaningful variable name.